backend/clap/src/main/java/zpi/ppea/clap/mappers/DetailedSubmissionMapper.java

```diff
@@ -0,0 +1,33 @@
+package zpi.ppea.clap.mappers;
+
+import data_store.DetailsSubmissionResponse;
+import zpi.ppea.clap.dtos.AppReportDto;
+import zpi.ppea.clap.dtos.DetailsSubmissionResponseDto;
+import zpi.ppea.clap.dtos.ProjectState;
+
+public class DetailedSubmissionMapper {
```

Comment:
perhaps we should write our own getters and setters?

How will this behave if we decide to add field/column to one of those structs, for example in app report - will this code produce an error/warning that not all fields are included/mapped?

Reply:
If there is a change in entity I have to change them manually. I have to remember about it.